plugins/messagelayer/parameters.go

```diff
@@ -29,12 +29,6 @@ type ParametersDefinition struct {
 
 // ManaParametersDefinition contains the definition of the parameters used by the mana plugin.
 type ManaParametersDefinition struct {
-	// EmaCoefficient1 defines the coefficient used for Effective Base Mana 1 (moving average) calculation.
```

Comment:
Did we want to remove the coefficient for mana1 as well? Or was it never actually used?

Reply:
It was never actually used!